ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/documentation/devices/DC1-POD1-LEAF2B.md

```diff
@@ -964,7 +964,13 @@ router bgp 65112.100
       rd evpn domain remote 172.16.110.5:50111
       route-target both 50111:50111
       route-target import export evpn domain remote 50111:50111
+      redistribute connected
```

Comment:
"redistribute connected" is valid only for IP VRFs. For the testing purpose let's use" redistribute router-mac system" line.